tests/e2e/release/cht-config/contact-edit.specs.js

```diff
@@ -0,0 +1,64 @@
+const utils = require('../../../utils');
+const commonElements = require('../../../page-objects/common/common.po.js');
+const contactPage = require('../../../page-objects/contacts/contacts.po.js');
+const uuid = require('uuid');
+const districtId = uuid.v4();
+const districtName = uuid.v4();
+const healthCenterId = uuid.v4();
+const healtchCenterName = uuid.v4();
+const personName = uuid.v4();
+const personId = uuid.v4();
+
+describe('Editing contacts with the CHT config', function() {
+  beforeAll(done => {
+    return Promise.all([utils.saveDocs(expectedDocs)])
```

Comment:
This doesn't need a `Promise.all` - you can just do `utils.saveDocs(expectedDocs).then(...)`